### PR TITLE
Updates for must-gather

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/22770

### CODE REVIEW

File: troubleshooting/must_gather.adoc [hunk 1]
```diff
@@ -25,33 +25,32 @@ See the following procedure to start using the `must-gather` command:
...
+. Add the {product-title} image that is used for gathering data and the directory. Run the following command, where you insert the image and the directory for the output:
 +
-*Note:* If you want to check your managed clusters, find the `gather-managed.log` file that is located in the the `cluster-scoped-resources` directory:
+----
+oc adm must-gather --image=registry.redhat.io/rhacm2/acm-must-gather-rhel8:v2.5.0 --dest-dir=<directory>
+----
 +
-
+*Note:* To receive a complete data set, the following `must-gather` command must be ran for the multicluster engine, as well:
 +
```
Comment (swopebe) [inline troubleshooting/must_gather.adoc]: This should be in the MCE docs; we need to keep RHACM and MCE separated.

Comment (swopebe) [reply troubleshooting/must_gather.adoc]: The multicluster engine operator (see that folder for the doc where this should go).

Comment (dockerymick) [reply troubleshooting/must_gather.adoc]: Ok, ,got it. So this note should most likely be removed?

**Comment by swopebe** [comment]
Though I do not think many are using it @dockerymick MCE has it's own here: https://github.com/stolostron/rhacm-docs/blob/2.6_stage/multicluster_engine/must_gather.adoc

If that is sufficient, then I think you can keep your doc RHACM-centric without changing the MCE doc.

**Review by swopebe — APPROVED** [review]
approving based on separation on MCE and RHACM references.

File: troubleshooting/must_gather.adoc [hunk 2]
```diff
@@ -25,11 +25,16 @@ See the following procedure to start using the `must-gather` command:
...
 . Learn about the `must-gather` command and install the prerequisites that you need at https://docs.openshift.com/container-platform/4.9/support/gathering-cluster-data.html[Gathering data about your cluster] in the {ocp} documentation.
 
-. Log in to your cluster. For the usual use-case, you should run the `must-gather` while you are logged into your _hub_ cluster. 
+. Add the {product-title} image that is used for gathering data and the directory. Run the following command, where you insert the image and the directory for the output:
++
+----
+oc adm must-gather --image=registry.redhat.io/rhacm2/acm-must-gather-rhel8:v2.5.0 --dest-dir=<directory>
+----
+
+. Log in to your cluster. For the usual use-case, you should run the `must-gather` while you are logged into your hub cluster.
```
Comment (dislbenn) [inline troubleshooting/must_gather.adoc]: @dockerymick should we include `Log in to your cluster` in step two? I can scenarios where people would attempt to run step two before logging into the cluster.

**Review by dislbenn — APPROVED** [review]

File: troubleshooting/must_gather.adoc (hunk 2 above)
Comment (dockerymick) [reply troubleshooting/must_gather.adoc]: @dislbenn To be sure that I am understanding your suggestion.  Add the instruction to "Log in.." should be moved to step 2? 

2. Log in to your cluster. Add the {product-title} image that is used for gathering data and the directory. Run the following command, where you insert the image and the directory for the output:

   ```
   oc adm must-gather --image=registry.redhat.io/rhacm2/acm-must-gather-rhel8:v2.5.0 --dest-dir=<directory>
   ```

3. For the usual use-case, you should run the `must-gather` while you are logged into your hub cluster.

Comment (dislbenn) [reply troubleshooting/must_gather.adoc]: Yes. That is what I would recommended.

**Pull request merged**